### ISSUE TITLE
Move FAQ to GitHub Pages

### ISSUE THREAD

**Issue by lossyrob** [locationtech/jts#122]
There was valuable information in the FAQ hosted at the now defunct tsusiasoftware.net. The wayback machine to the rescue: https://web.archive.org/web/20160123233029/http://tsusiatsoftware.net:80/

This FAQ should be resurrected and placed into the GitHub pages.

Under the `Documentation` header of the main page, there should be a link to the FAQ. The FAQ should be a new page with the same style as the current GitHub pages. Bonus points for adding the proper links from the header down to each individual question.

This site is a [GitHub Project Page](https://help.github.com/articles/user-organization-and-project-pages/#project-pages), and the site's code is in the `gh-pages` branch of this repository. Please make a pull request to that branch for this feature.

**Comment by moovida** [comment]
Hi, I would love to jump in for this if it is about migrating the old FAQ to a new WIKI page here. 
But while I have no problems to find the old FAQ, I am not sure I understand where the new one should go. You are mentioning the gh-pages, while the user and developers docs are in the "normal" github wiki pages.

**Comment by lossyrob** [comment]
Hi Andrea,

Where is the wiki you are mentioning?

The docs I am talking about create this site:
http://locationtech.github.io/jts/

And are located at the `gh-pages` branch of jts:
https://github.com/locationtech/jts/tree/gh-pages

The wiki I thought you were mentioning seem empty:
https://github.com/locationtech/jts/wiki, so I'm curious about where you're
seeing docs in the 'normal' github wiki pages. If there are other docs
floating around out there, we should try to consolidate them.

Thanks,
Rob

**Comment by moovida** [comment]
Aha, now I see where it is.

I was looking under Documentation in the main page and for example:

```
Installing and using JTS is described in the User Guide.
```
leads to a wiki page: https://github.com/locationtech/jts/blob/master/USING.md


Ok, so it seems that it is all about migrating some stuff into the plain html of the gh-pages site (I do not see any sphinx or docs engine involved, right?).

**Comment by dr-jts** [comment]
Technically I think the MD pages in the repo are not part of the Github wiki.  (But I could be wrong, since GH has quite a bit of implicit magic behind it).

And yes, this is about migrating the HTML of the FAQ into the gh-pages website.  So need to do things like:
- fix the CSS reference, and verify formatting looks ok
- add link from the main website page
- fix any links in the FAQ which need it

Nice-to-haves:
- Create anchor links for each entry, and link the table of contents to them
- Turn any JTS class references into links into the Javadoc

**Comment by moovida** [comment]
Ok, then count me in. I will report back when I have the PR to discuss.

**Comment by moovida** [comment]
Ok, the page is brought in with PR https://github.com/locationtech/jts/pull/127

The only thing I think is left out, is the ```fix the CSS reference, and verify formatting looks ok```, which I am not sure what it is about. The formatting looks fine. I would prefer the <code> tag to have a larger font, but this can be discussed.

I notice that merging is not accepted due to ```The following users do not have valid ECAs```. I apologize for that, I was in the wrong assumption that I did a long long time ago, as I was a uDig committer when the locationtech move happened. Not sure what is the best way to handle this.

**Comment by lossyrob** [comment]
@moovida the ECA might have expired. You'll have to go to https://www.eclipse.org/legal/ECA.php and sign it again, with an eclipse account that has the same email as your git email. You'll also have to sign-off on the commits, as I commented on the PR. Once that goes through, I believe this PR is ready to be merged and this issue closed. Thank you!
